scripts/config-dhcp.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+#Check if isc-dhcp-server exists, if not install it.
+while true
+do
+	if dpkg-query -W -f='${Status} ${Version}' isc-dhcp-server | grep 4.3; then
+		echo "Already installed"
+		sudo dpkg --configure -a
+		break 
+	else
+		echo "Installing isc-dhcp-server"
```

Comment:
This needs to be removed. It has to be done in one place and one place only, the update process.